channels/fr.m3u

```diff
@@ -309,42 +285,36 @@ http://f24hls-i.akamaihd.net/hls/live/520845/F24_ES_HI_HLS/master.m3u8
+http://mv2.tvfrancophonie.org/live/Stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="Francophonie24.fr" tvg-name="Francophonie24" tvg-country="FR" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Francophonie24_275x275.png?raw=true" group-title="",Francophonie24
 https://5421175365ea3.streamlock.net/live/smil:switch.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="FTVHD.fr" tvg-name="FTV HD" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",FTV HD
-https://fash1043.cloudycdn.services/slive/_definst_/ftv_ftv_midnite_k1y_27049_midnite_secr_108_hls.smil/chunklist_b4700000_t64MTA4MHA=.m3u8
 #EXTINF:-1 tvg-id="FTVNews.fr" tvg-name="FTV News" tvg-country="FR" tvg-language="" tvg-logo="" group-title="News",FTV News
 http://210.61.56.23/hls/ftvtv/index.m3u8
-#EXTINF:-1 tvg-id="FTVTeens.fr" tvg-name="FTV Teens" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",FTV Teens
-https://fash1043.cloudycdn.services/slive/_definst_/ftv_ftv_paris_pg_4dg_27027_paris_pg18_188_hls.smil/chunklist_b1400000_t64NDgwcA==.m3u8
 #EXTINF:-1 tvg-id="GenerationsTV.fr" tvg-name="Generations TV" tvg-country="FR" tvg-language="" tvg-logo="https://i.imgur.com/daUGyPl.jpg" group-title="",Generations TV
-https://edge10.vedge.infomaniak.com/livecast/ik:generation-tv/manifest.m3u8?spark=28dc12a7-48b5-4d36-9b5b-c2deb9b70bcb&token=
-#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
-http://str81.creacast.com/grandlilletv/smil:grandlilletv.smil/chunklist_w1118883208.m3u8
-#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",Grand Lille TV
-https://live.creacast.com/grandlilletv/smil:grandlilletv.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="GrandLilleTVHD.fr" tvg-name="Grand Lille TV HD" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",Grand Lille TV HD
+https://edge10.vedge.infomaniak.com/livecast/ik:generation-tv/manifest.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
+http://str81.creacast.com/grandlilletv/smil:grandlilletv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
+https://live.creacast.com/grandlilletv/smil:grandlilletv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
 http://str81.creacast.com/grandlilletv/high/playlist.m3u8
```

Comment:
Non-adaptive link of the the source above?

Reply:
Good catch.